README.md

```diff
@@ -52,6 +52,8 @@ Make sure you have setup Rust environment (>= 1.64).
 - Build wasm
   - `yarn build-wasm`
 
+For webstorm users, you may need to run `yarn build-wasm` in terminal rather than webstorm's build-in run tool.
```

Comment:
this is not the case. we doing local setup you need to build wasm always, it's mention above